### PR TITLE
feat[3.5] pharmacist profile

### PR DESCRIPTION
**Pull request by Pufke**
- [x] Fetch all examinations for Pharmacist and show examinations on calendar
- [x] Fetch Pharmacist personal data and on Profil page

### CODE REVIEW

File: frontend-vue-js/src/plugins/router.js [hunk 1]
```diff
@@ -67,6 +68,11 @@ const routes = [
                 component: PharmacistLanding,
                 name: 'pharmacist',
                 path: ''
+            },
+            {
+                component: PharmacistProfile,
+                name: 'pharmacistProfile',
+                path: '/pharmacistProfile'
```
Comment (milana-todorovic) [inline frontend-vue-js/src/plugins/router.js:75]: This path will already have pharmacist in it so it's currently '/pharmacist/pharmacistProfile', wouldn't just '/pharmacist/profile' make more sense?

Comment (Pufke) [reply frontend-vue-js/src/plugins/router.js:75]: Of sure, nice, i will fix that

**Comment by Pufke** [comment]
[(2) 3.5 Profil farmaceuta/dermatologa](https://trello.com/c/K0f9eubZ/45-2-35-profil-farmaceuta-dermatologa)

**Comment by Pufke** [comment]
[3.36 Grafički prikaz radnog kalendara](https://trello.com/c/uAiKBQDF/54-336-grafi%C4%8Dki-prikaz-radnog-kalendara)

File: frontend-vue-js/src/plugins/router.js (hunk 1 above)
Comment (Pufke) [reply frontend-vue-js/src/plugins/router.js]: Fixed

**Comment by Pufke** [comment]
email: pharmacist2@gmail.com

image

**Comment by Pufke** [comment]
email: pharmacist@gmail.com
password: pharmacist
image

File: frontend-vue-js/src/plugins/router.js [hunk 2]
```diff
@@ -67,6 +68,11 @@ const routes = [
                 component: PharmacistLanding,
                 name: 'pharmacist',
                 path: ''
+            },
+            {
+                component: PharmacistProfile,
+                name: 'profile',
+                path: '/pharmacist/profile'
```
Comment (milana-todorovic) [inline frontend-vue-js/src/plugins/router.js:75]: This is a child component, the parent's path is already '/pharmacist'. You should define the child's path as just 'profile' (with no slash) to get this path.

Comment (Pufke) [reply frontend-vue-js/src/plugins/router.js]: Fixed

**Review by milana-todorovic — APPROVED** [review]

**Pull request merged**